utils/tfhe-versionable/examples/manual_impl.rs

```diff
@@ -2,11 +2,10 @@
 
 use std::convert::Infallible;
 
-use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use tfhe_versionable::{Unversionize, UnversionizeError, Upgrade, Versionize, VersionizeOwned};
 
-struct MyStruct<T: Default> {
+struct MyStruct<T> {
```

Comment:
is default removed because is causes problem or it was not needed / its a style change ?

Reply:
It was not needed, but it was before. The change that allowed this was done in this pr: https://github.com/zama-ai/tfhe-rs/pull/1551.

In that case it doesn't change anything but sometimes you don't want to have a too restrictive bound, you want to be able to create MyStruct with any T but only be able to unversionize/upgrade if `T: Default`. 

A more realistic example is in tfhe-rs with container, we might require `C: ContainerMut` for some upgrade implementation (and so needed for `Unversionize`) but `C: Container` is sufficient for `Versionize`